src/chart/bar/BarView.js

```diff
@@ -127,15 +128,30 @@ export default echarts.extendChartView({
 
         var roundCap = seriesModel.get('roundCap', true);
 
+        var drawBackground = seriesModel.get('showBackground');
+        var backgroundModel = seriesModel.getModel('backgroundStyle');
+
+        var bgEls = [];
+        var oldBgEls = this._backgroundEls || [];
+
+        var oldLargeBackgroundEl = group.childOfName('largeBackground');
```

Comment:
It might not be a efficient way if using `childOfName` since all of the bar `el` are added to `group`.
I think it is OK to mount it to `this`.